packages/checkbox/checkbox.scss

```diff
@@ -64,6 +67,54 @@ $checkbox-box-default-color--inverted: jkl.$color-svaberg;
     }
 }
 
+@mixin checkbox-error-background($box-size) {
+    /**
+     * Denne remsa med CSS trenger kanskje litt forklaring.
```

Comment:
Samme kommentar som på radio button

Reply:
På checkbox kan vi vel forresten gå over til å bruke `outline` for fokusringen, siden den ikke trenger avrundede hjørner. Det er hakket mer accessible, og vi slipper å bruke `::before`-elementet i det hele tatt 😊 Trenger ikke nødvendigvis gjøre det i denne PRen, men kom på det siden vi uansett ser på elementet det gjelder.